test/shlibloadtest.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Where dual copyright means that there are 2 people having a copyright and both need to provide a license. Which is different from a dual license where 1 copyright holder makes it available under 2 licenses.